.github/workflows/ci_test-full.yml

```diff
@@ -18,11 +18,17 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-18.04, windows-2019, macOS-10.15]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.8, 3.9]
```

Comment:
why's 3.7 removed?

Reply:
our test started to be quite large and I do not expect that there is anything on pure python that would work on both versions around 3.6 and 3.8 and was not working in 3.7

Reply:
@carmocca suggestion?

Reply:
Sounds good to me.

Lets remove Python 3.6 support in 8 months (its end-of-life)

https://endoflife.date/python

Reply:
ol, then we set 3.7 as min, that is a good point...
well py.36 is one of the most used python version for us...
https://pypistats.org/packages/pytorch-lightning